app/renderer/components/navbar.js

```diff
@@ -26,7 +26,7 @@ module.exports = {
                 <tap-target v-bind:activates="'tap-edit'" v-bind:title="'Add some tasks !'" v-bind:description="'By pressing this button you can add new tasks to your list below.'"></tap-target>
                 <ul class="right navbar-buttons">
                     <li><a id="tap-edit" v-on:click="toggleEdit" v-bind:class="{'edit-button-active': editMode}" class="edit-button"><i class="material-icons unselectable-text">mode_edit</i></a></li>
-                    <li><a class="navbar-menu-button" data-activates='navbar-menu' data-gutter="30"><i class="material-icons small unselectable-text">menu</i></a></li>
+                    <li><a id="navBarMenu" class="navbar-menu-button" data-activates='navbar-menu' data-gutter="30"><i class="material-icons small unselectable-text">menu</i></a></li>
```

Comment:
I'm thinkusing the id `navBarMenu` is misleading. Keep in mind that this is not the actual menu, but the button.

If you need an id, we could turn the class `navbar-menu-button` into an id (making sure it is not being used anywhere else) and modifying the related css